src/modm/platform/clock/rp/clocks.hpp.in

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2013, Kevin Läufer
+ * Copyright (c) 2014-2017, Niklas Hauser
+ * Copyright (c) 2017, Fabian Greif
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef MODM_LPC_CLOCK_HPP
+#define MODM_LPC_CLOCK_HPP
+
+#include <stdint.h>
+#include <modm/platform/device.hpp>
+#include <modm/architecture/interface/delay.hpp>
+#include <hardware/structs/clocks.h>
+#include <hardware/structs/pll.h>
+#include <hardware/structs/resets.h>
+
+namespace modm
+{
+	namespace platform
+	{
```

Comment:
merge into namespace modm::platform and remove indentation to see more of the code.